examples/digits_dann_lightn/main.py

```diff
@@ -69,7 +69,9 @@ def main():
             gpus=args.gpus,
             logger=False,  # logger,
             # weights_summary='full',
-            fast_dev_run=True,  # True,
+            # limit_train_batches=0.005,
+            # limit_val_batches=0.06,
+            # limit_test_batches=0.06,
```

Comment:
Adding the above three lines for what

Reply:
Although `fast_dev_run` is very useful, it disables `checkpoint` and `early stopping` callbacks. This may cause bugs related to callbacks. See https://pytorch-lightning.readthedocs.io/en/latest/common/debugging.html#fast-dev-run.

An alternative method is adding these three lines to only reduce the size of the data. We can test the code on the smallest dataset before running it on the whole dataset. We can remove or comment `fast_dev_run` if using this method.

Reply:
Good to know. Some light comments may help others. Thanks!